Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeViewController.swift

```diff
@@ -262,15 +263,21 @@ class CreateSafeViewController: UIViewController, UITableViewDelegate, UITableVi
         show(selectNetworkVC, sender: self)
     }
 
-    func selectOwnerRow(_ rowIndex: Int) {
-        guard rowIndex == uiModel.owners.count else { return }
-        addOwner()
+    func selectOwnerRow(tableView: UITableView, indexPath: IndexPath) {
+        guard indexPath.row == uiModel.owners.count else { return }
+        addOwner(tableView: tableView, indexPath: indexPath)
     }
 
-    func addOwner() {
+    func addOwner(tableView: UITableView, indexPath: IndexPath) {
         let picker = SelectAddressViewController(chain: uiModel.chain, presenter: self) { [weak self] address in
             self?.uiModel.addOwnerAddress(address)
         }
+
+        if let popoverPresentationController = picker.popoverPresentationController {
+            // TODO what view to use?
+            popoverPresentationController.sourceView = tableView
```

Comment:
you can try to use tableView.cellForRow(at: indexPath) to get the source view as a row